### DISCUSSION TITLE
What happens to the account of a person who designates a successor and actually dies?

### DISCUSSION THREAD

**Discussion by kaizawa97** [community/community#34872] [Other Feature Feedback, Questions, & Ideas] [answered]
H. I'm kaizawa97.
I have one question for everyone.

What happens to the account of a person who designates a successor and dies?
Does the GitHub policy say that the successor will access the public repositories, but the account itself will remain?

Thank you.

**Comment by Anmol-Baranwal** [comment, marked as answer]
As a successor, they will have permission to:
- Archive your public repositories.
- Transfer your public repositories to their own user-owned account.
- Transfer your public repositories to an organization where they can create repositories.

**Note:** Successors cannot log into your account.

An appointed successor can manage your public repositories after presenting a death certificate then waiting for 7 days or presenting an obituary then waiting for 21 days. For more information, see "[GitHub Deceased User Policy](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-deceased-user-policy)."

**Reply by kaizawa97** [reply]
Dear Anmol-Baranwal-san.

Are accounts of deceased persons deleted?
Or will it remain?

Thank you.

**Reply by Anmol-Baranwal** [reply]
As I mentioned, they cannot log into the account, so yes the account will remain.

**Reply by Anmol-Baranwal** [reply]
@kaizawa97 please mark this as an answer if you have no further queries so that others can know that the question has been answered.

**Reply by kaizawa97** [reply]
Dear Anmol-Baranwal-san.

Thank you for your kindness.

Best kaizawa97.

**Reply by karlhorky** [reply]
> For more information, see "[GitHub Deceased User Policy](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-deceased-user-policy)."

@Anmol-Baranwal your link above is a bit misleading, because it doesn't contain the information about the 7 days / 21 days timeline.

This link has the timeline - maybe that link should also be added as the source of your content:

- https://docs.github.com/en/repositories/creating-and-managing-repositories/access-to-repositories#about-successors
